src/FakeQueryBuilder.php

```diff
@@ -98,15 +98,15 @@ public function whereNotNull($columns, $boolean = 'and')
         return $this;
     }
 
-    public function whereBetween($column, array $values, $boolean = 'and', $not = false)
+    public function whereBetween($column, iterable $values, $boolean = 'and', $not = false)
     {
         $this->recordedWhereBetween[] = [$column, $values];
 
         return $this;
 
     }
 
-    public function whereNotBetween($column, array $values, $boolean = 'and')
+    public function whereNotBetween($column, iterable $values, $boolean = 'and')
```

Comment:
this change for compatible with the parent Builder  class I got exception in running